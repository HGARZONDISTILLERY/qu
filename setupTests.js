import { QueryClient, QueryClientProvider } from 'react-query';
import { render } from '@testing-library/react';

const queryClient = new QueryClient();

const customRender = (ui, options) =>
  render(ui, { wrapper: ({ children }) => <QueryClientProvider client={queryClient}>{children}</QueryClientProvider>, ...options });

export * from '@testing-library/react';
export { customRender as render };
