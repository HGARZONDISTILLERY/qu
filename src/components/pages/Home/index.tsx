import { Box } from "@mui/system";
import TemplateJoke from "../../templates/TemplateJoke";

import "./styles.css";

const Home = () => {
  return (
    <>
      <Box component="header">
        <h1>The joke show</h1>
      </Box>
      <TemplateJoke />
    </>
  );
};

export default Home;
