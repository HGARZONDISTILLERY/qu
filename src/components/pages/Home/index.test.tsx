import Home from '.';
import {render} from '../../../../setupTests'

test('Home component renders correctly', () => {
  const tree = render(<Home />);
  expect(tree).toMatchSnapshot();
});
