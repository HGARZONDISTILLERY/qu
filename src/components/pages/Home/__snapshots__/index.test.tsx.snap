// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home component renders correctly 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <header
        class="MuiBox-root css-0"
      >
        <h1
          class="MuiTypography-root MuiTypography-h1 joke-show-title css-o2w69a-MuiTypography-root"
        >
          The joke show
        </h1>
      </header>
      <div
        class="MuiBox-root css-13800us"
      >
        <span
          style="width: 165px; font-size: 0px; display: inline-block;"
        >
          <span
            style="display: inline-block; background-color: rgb(188, 16, 1); width: 15px; height: 15px; margin: 20px; border-radius: 100%; animation-fill-mode: both; animation: react-spinners-GridLoader-grid 1.445250190510842s 0.6452501905108419s infinite ease;"
          />
          <span
            style="display: inline-block; background-color: rgb(188, 16, 1); width: 15px; height: 15px; margin: 20px; border-radius: 100%; animation-fill-mode: both; animation: react-spinners-GridLoader-grid 0.8560330895376734s 0.05603308953767344s infinite ease;"
          />
          <span
            style="display: inline-block; background-color: rgb(188, 16, 1); width: 15px; height: 15px; margin: 20px; border-radius: 100%; animation-fill-mode: both; animation: react-spinners-GridLoader-grid 0.9118207260158969s 0.11182072601589693s infinite ease;"
          />
          <span
            style="display: inline-block; background-color: rgb(188, 16, 1); width: 15px; height: 15px; margin: 20px; border-radius: 100%; animation-fill-mode: both; animation: react-spinners-GridLoader-grid 1.2172527747045945s 0.4172527747045944s infinite ease;"
          />
          <span
            style="display: inline-block; background-color: rgb(188, 16, 1); width: 15px; height: 15px; margin: 20px; border-radius: 100%; animation-fill-mode: both; animation: react-spinners-GridLoader-grid 0.9750925892113991s 0.17509258921139914s infinite ease;"
          />
          <span
            style="display: inline-block; background-color: rgb(188, 16, 1); width: 15px; height: 15px; margin: 20px; border-radius: 100%; animation-fill-mode: both; animation: react-spinners-GridLoader-grid 1.0600468332783426s 0.2600468332783427s infinite ease;"
          />
          <span
            style="display: inline-block; background-color: rgb(188, 16, 1); width: 15px; height: 15px; margin: 20px; border-radius: 100%; animation-fill-mode: both; animation: react-spinners-GridLoader-grid 1.505027502413499s 0.7050275024134991s infinite ease;"
          />
          <span
            style="display: inline-block; background-color: rgb(188, 16, 1); width: 15px; height: 15px; margin: 20px; border-radius: 100%; animation-fill-mode: both; animation: react-spinners-GridLoader-grid 0.6781760841661092s -0.1218239158338908s infinite ease;"
          />
          <span
            style="display: inline-block; background-color: rgb(188, 16, 1); width: 15px; height: 15px; margin: 20px; border-radius: 100%; animation-fill-mode: both; animation: react-spinners-GridLoader-grid 0.9594808204401146s 0.15948082044011458s infinite ease;"
          />
        </span>
      </div>
    </div>
  </body>,
  "container": <div>
    <header
      class="MuiBox-root css-0"
    >
      <h1
        class="MuiTypography-root MuiTypography-h1 joke-show-title css-o2w69a-MuiTypography-root"
      >
        The joke show
      </h1>
    </header>
    <div
      class="MuiBox-root css-13800us"
    >
      <span
        style="width: 165px; font-size: 0px; display: inline-block;"
      >
        <span
          style="display: inline-block; background-color: rgb(188, 16, 1); width: 15px; height: 15px; margin: 20px; border-radius: 100%; animation-fill-mode: both; animation: react-spinners-GridLoader-grid 1.445250190510842s 0.6452501905108419s infinite ease;"
        />
        <span
          style="display: inline-block; background-color: rgb(188, 16, 1); width: 15px; height: 15px; margin: 20px; border-radius: 100%; animation-fill-mode: both; animation: react-spinners-GridLoader-grid 0.8560330895376734s 0.05603308953767344s infinite ease;"
        />
        <span
          style="display: inline-block; background-color: rgb(188, 16, 1); width: 15px; height: 15px; margin: 20px; border-radius: 100%; animation-fill-mode: both; animation: react-spinners-GridLoader-grid 0.9118207260158969s 0.11182072601589693s infinite ease;"
        />
        <span
          style="display: inline-block; background-color: rgb(188, 16, 1); width: 15px; height: 15px; margin: 20px; border-radius: 100%; animation-fill-mode: both; animation: react-spinners-GridLoader-grid 1.2172527747045945s 0.4172527747045944s infinite ease;"
        />
        <span
          style="display: inline-block; background-color: rgb(188, 16, 1); width: 15px; height: 15px; margin: 20px; border-radius: 100%; animation-fill-mode: both; animation: react-spinners-GridLoader-grid 0.9750925892113991s 0.17509258921139914s infinite ease;"
        />
        <span
          style="display: inline-block; background-color: rgb(188, 16, 1); width: 15px; height: 15px; margin: 20px; border-radius: 100%; animation-fill-mode: both; animation: react-spinners-GridLoader-grid 1.0600468332783426s 0.2600468332783427s infinite ease;"
        />
        <span
          style="display: inline-block; background-color: rgb(188, 16, 1); width: 15px; height: 15px; margin: 20px; border-radius: 100%; animation-fill-mode: both; animation: react-spinners-GridLoader-grid 1.505027502413499s 0.7050275024134991s infinite ease;"
        />
        <span
          style="display: inline-block; background-color: rgb(188, 16, 1); width: 15px; height: 15px; margin: 20px; border-radius: 100%; animation-fill-mode: both; animation: react-spinners-GridLoader-grid 0.6781760841661092s -0.1218239158338908s infinite ease;"
        />
        <span
          style="display: inline-block; background-color: rgb(188, 16, 1); width: 15px; height: 15px; margin: 20px; border-radius: 100%; animation-fill-mode: both; animation: react-spinners-GridLoader-grid 0.9594808204401146s 0.15948082044011458s infinite ease;"
        />
      </span>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
